File: solutions/ruby/darts/1/darts.rb
class Darts
  def initialize(x, y)
    @x_coordinate = x
    @y_coordinate = y
  end
  def score()
    distance = Math.hypot(@x_coordinate, @y_coordinate)
    distance < 1 ? 10 : distance < 5 ? 5 : distance < 10 ? 1 : 0;
  end
end

File: solutions/java/darts/1/src/main/java/Darts.java
class Darts {
    double x, y;

    Darts(double x, double y) {
        this.x = x;
        this.y = y;
    }

    int score() {
        double distance = Math.hypot(x, y);
        return distance <= 1 ? 10 : distance <= 5 ? 5 : distance <= 10 ? 1 : 0;
    }

}

